scl/paloalto/panos.conf

```diff
@@ -0,0 +1,97 @@
+#############################################################################
+# Copyright (c) 2020 Balabit
+# created by mileek@gmail.com
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
```

Comment:
The commit check also fails on this PR. There is a strict format that is asserted in the CI.

You can get general idea from: https://github.com/syslog-ng/syslog-ng/blob/06df1d712cf0608845eba0662e2cba7f85703af8/CONTRIBUTING.md#L88

In your case, the module part is missing. For example, the first line could look like:
```
panos-parser: new source as SCL
```

Also, this is a parser, isn't it? You could write parser instead of source in the commit message.

Reply:
Unfortunately that blog is outdated. commit-check.sh is introduced one year later.

@czanik could you update the commit message example the referred blogpost? We have a hard requirement on the commit messages since then. For example: the first line might be:
```
scl: new linux-audit() source
```

Reply:
Sure. I put it on my ToDo list.